### PR TITLE
[chains] - testnet added

### PR DESCRIPTION
**Pull request by jnsdls**

### CODE REVIEW

File: packages/chains/chains/84531.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      512,
+      256,
+      128,
+      64,
+      32,
+      16
+    ]
+  },
+  "testnet": true,
+  "slug": "base-testnet-goerli"
```
Comment (joaquim-verges) [inline packages/chains/chains/84531.ts:33]: @jnsdls can we just do `base-goerli` instead of `base-testnet-goerli` ?

Comment (jnsdls) [reply packages/chains/chains/84531.ts:33]: sure if I add another custom rule lol

**Pull request closed without merging**